prov/shm/src/smr_init.c

```diff
@@ -157,6 +159,7 @@ struct util_prov smr_util_prov = {
 SHM_INI
 {
 
+	old_action = (struct sigaction *)calloc(SIGRTMIN, sizeof(struct sigaction));
```

Comment:
We need to check to failure here.  Use sizeof(*old_action).  No need for casting calloc return.

Reply:
Updated